### ISSUE TITLE
npm test doesn't pass on master because typings

### ISSUE THREAD

**Issue by usergenic** [Polymer/web-component-tester#552]
```
npm test

> web-component-tester@6.0.0 test /Users/brendanb/src/github.com/polymer/web-component-tester
> tsc && gulp test


15     interface Gulp extends Orchestrator {
                 ~~~~

node_modules/@types/gulp/index.d.ts(15,15): error TS2430: Interface 'Gulp' incorrectly extends interface 'Orchestrator'.
  Types of property 'task' are incompatible.
    Type 'AddMethod' is not assignable to type '{ (name: string): Task; (name: string, fn: TaskFunc): void; (name: string, dep: string[], fn: Tas...'.
      Type 'Orchestrator' is not assignable to type 'Task'.
        Property 'fn' is missing in type 'Orchestrator'.

npm ERR! Test failed.  See above for more details.
```

**Comment by maxknee** [comment]
it's getting me
https://github.com/Polymer/web-component-tester/pull/554

but it doesn't happen locally... 
osx 10.11.6
```typescript global is Version 2.3.3
gulp: 3.9.1```

**Comment by maxknee** [comment]
@usergenic Fixed! updated `@types/gulp` to `4.0.3`
